apps/web/modules/settings/my-account/appearance-view.tsx

```diff
@@ -304,6 +304,7 @@ const AppearanceView = ({
               description={t("bookerlayout_user_settings_description")}
               isDisabled={isBookerLayoutFormSubmitting || !isBookerLayoutFormDirty}
               isLoading={mutation.isPending}
+              user={user}
```

Comment:
can you give some context on this change ?

Reply:
https://github.com/calcom/cal.com/pull/16711#discussion_r1770839258